lib/modules/onc_program_procedure.yml

```diff
@@ -0,0 +1,1219 @@
+procedure:
+  - section: Paragraph (g)(10)(iii) - Application registration
+    steps:
+      - group: Application Registration
+        index: 1
+        SUT: |
+          The health IT developer demonstrates the Health IT Module
+          supports application registration with an authorization server for
+          the purposes of Electronic Health Information (EHI) access for
+          single patients, including support for application registration
+          functions to enable authentication and authorization in §
+          170.315(g)(10)(v)
+        TLV: |
+          The tester verifies the Health IT Module supports application
+          registration with an authorization server for the purposes of EHI
+          access for single patients, including support for application
+          registration functions to enable authentication and authorization in
+          § 170.315(g)(10)(v).
+        inferno_supported: no
+        inferno_notes: |
+          Application registration is considered out of scope for Inferno
+          Program Edition because it cannot be done in an automated fashion.
+          Inferno Program Edition may in the future include Dynamic Registration
+          functionality for those systems that support that standard.  It may
+          also include an input field to allow the tester to document that this
+          has been performed.
+      - index: 2
+        SUT: |
+          The health IT developer demonstrates the Health IT Module
+          supports application registration with an authorization server for
+          the purposes of EHI access for multiple patients, including support
+          for application registration functions to enable authentication and
+          authorization in § 170.315(g)(10)(v).
+        TLV: |
+          The tester verifies the Health IT Module supports application
+          registration with an authorization server for the purposes of EHI
+          access for multiple patients including support for application
+          registration functions to enable authentication and authorization in
+          § 170.315(g)(10)(v).
+        inferno_supported: no
+        inferno_notes: |
```

Comment:
There isn't as strong a case for Dynamic Registration for Backend Services.  But since it is built on OAuth 2.0, it could be used.  I copy/pasted this from above, but think it would make sense to just remove it.